### DISCUSSION TITLE
[Bug?] Hosted Checkout doesn't alert that a product is out of stock

### DISCUSSION THREAD

**Discussion by marnauortega** [swellstores/community#378] [Help] [answered]
When tracking the inventory of a product, I can set it to 0 and disable the "allow purchase when out of stock":

![Captura de Pantalla 2023-09-17 a les 21 52 02](https://github.com/swellstores/community/assets/37710726/884e33ff-d5bd-417c-91eb-3a0aa79c545d)

I thought this would trigger some sort of message on the checkout explaining that the purchase cannot be done because the product is out of stock. What happens instead is just that on the last step, when clicking on "complete order", the page reloads and the purchase doesn't complete, but the customer gets no explanation:

https://github.com/swellstores/community/assets/37710726/fcef2afc-8a4d-4ec9-840d-b9075360cb18

**Comment by SusannaSwell** [comment, marked as answer]
hello @marnauortega  was able to reproduce your issue only when there is 0 to pay for an order.
If there is the price to pay, a message about not available product is displayed after clicking on submit button

<img width="1178" alt="Screenshot 2023-09-18 at 11 32 59" src="https://github.com/swellstores/community/assets/79216957/98b8ed0b-cdb3-453c-9636-6a24c937e658">

will create a report in our internal system for fixing this case. will update you once I have the detaills.

**Reply by marnauortega** [reply]
Ah, great. I put the 0 price because I was testing. But then it's no big deal. Thanks a lot for your help!

**Comment by steven-k1** [comment]
A fix has been deployed for this issue, and a message alerting that there is no stock when there is 0 will now be displayed.
